<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtsWidget</class>
 <widget class="QWidget" name="AtsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATSTest</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>80</y>
     <width>421</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>监听协议帧</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>ATS-&gt;VOBC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ATS心跳信息帧              周期（ms）：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ats_vobc_heart_time"/>
      </item>
      <item>
       <widget class="QPushButton" name="ats_vobc_heart">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ATO命令信息帧                                        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ats_vobc_cmd">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>ATS-&gt;CI</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>ATS时间信息               周期(ms)：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ats_ci_time_time"/>
      </item>
      <item>
       <widget class="QPushButton" name="ats_ci_time">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>站场操控命令                                         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ats_ci_opcmd">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ATS运行状态信息           周期(ms)：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ats_ci_run_time"/>
      </item>
      <item>
       <widget class="QPushButton" name="ats_ci_run">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>ATS-&gt;ZC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ATS时间信息              周期(ms)：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ats_zc_time_time"/>
      </item>
      <item>
       <widget class="QPushButton" name="ats_zc_time">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>TSR验证命令帧                                        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tsr_sure">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>TSR执行命令帧                                        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tsr_opcmd">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>上电TSR确认帧                                        </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tsr_power">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>237</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>报警数据</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLineEdit" name="dev_local"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>alarm_subtype</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>alarm_level</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>alarm_code</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="alarm_subtype"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="alarm_type"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="ats_mss_button">
      <property name="text">
       <string>发送</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>alarm_type</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="alarm_code"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="alarm_level"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="dev_type"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>dev_id</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>dev_local</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="dev_id"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>dev_type</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="sys_id"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>sys_id</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBoxVobc">
      <property name="text">
       <string>转发VOBC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>20</y>
     <width>492</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ip"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="port"/>
    </item>
    <item>
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>传输协议：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Basic UDP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RSSPI</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
